orderbook/src/main.rs

```diff
@@ -40,6 +42,15 @@ struct Arguments {
 
     #[structopt(long, env = "FEE_DISCOUNT_FACTOR", default_value = "1")]
     fee_discount_factor: f64,
+
+    /// The minimum amount of time an order has to be valid for.
+    #[structopt(
+        long,
+        env = "MIN_ORDER_VALIDITY_PERIOD",
+        default_value = "120",
```

Comment:
nit (and good that this is configurable): I'd personally say 60s would be sufficient in case a market maker wanted to specifically place a counter-order against an order they found in the orderbook.

Reply:
Changed.